src/index.html

```diff
@@ -1,16 +1,59 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+
+  <body class="page__body">
+
+    <div class="container">
+      <!-- #region formbig-->
+      <form
+        action="post"
+        class="form"
+        data-qa="big"
+        >
+        <button
+          id="icon-search-big"
+          class="icon-search icon-search--size-big form__search-button-big"
+          type="submit"
+          name="buttom search"
+        ></button>
+
+        <input
+          id="input-search"
+          class="input-search input-search--size-big form__search-input-big"
+          data-qa="keypress"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </form>
+      <!-- #endregion formsmall -->
+
+      <form
+        action="post"
+        class="form"
```

Comment:
Missing Block parent:

Same naming as before, should contain parent block class here also: class="container__form form"